lib/vmdb/loggers/api_logger.rb

```diff
@@ -0,0 +1,39 @@
+require_relative "multicast_logger"
```

Comment:
@Fryguy yeah, that doesn't seem to work for me - presumably because `MulticastLogger` is toplevel? Gives me:


```
NameError: uninitialized constant Vmdb::Loggers::MulticastLogger
  /home/tim/src/manageiq/lib/vmdb/loggers/api_logger.rb:3:in `<module:Loggers>'
  /home/tim/src/manageiq/lib/vmdb/loggers/api_logger.rb:2:in `<module:Vmdb>'
  /home/tim/src/manageiq/lib/vmdb/loggers/api_logger.rb:1:in `<top (required)>'
  /home/tim/src/manageiq/lib/vmdb/loggers.rb:90:in `block in <top (required)>'
  /home/tim/src/manageiq/lib/vmdb/loggers.rb:90:in `each'
  /home/tim/src/manageiq/lib/vmdb/loggers.rb:90:in `<top (required)>'
  /home/tim/src/manageiq/lib/vmdb/logging.rb:1:in `<top (required)>'
  /home/tim/src/manageiq/config/application.rb:108:in `<class:Application>'
  /home/tim/src/manageiq/config/application.rb:35:in `<module:Vmdb>'
  /home/tim/src/manageiq/config/application.rb:34:in `<top (required)>'
  /home/tim/src/manageiq/config/environment.rb:2:in `require'
  /home/tim/src/manageiq/config/environment.rb:2:in `<top (required)>'
```